lib/syskit/dynamic_port_binding.rb

```diff
@@ -319,6 +327,10 @@ def initialize(port)
                 @port = port
             end
 
+            def current_selection_valid?(port)
```

Comment:
there are two version of current_selection_valid...

Reply:
It depends on the resolver, for the ComponentPortResolver and CompositionChildPortResolver, I used the condition in the update, as for the CompositionChildPortResolver, I checked whether the matcher found the provided port